packages/teleport/src/Databases/ConnectDatabase/ConnectInstructions/ConnectInstructions.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2021 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Text, Box, ButtonSecondary, Link } from 'design';
+import { DialogContent, DialogFooter } from 'design/Dialog';
+import cfg from 'teleport/config';
+import { DbProtocol } from 'teleport/services/databases';
+import TextSelectCopy from 'teleport/components/TextSelectCopy';
+
+export default function ConnectInstructions({
+  user,
+  clusterId,
+  onClose,
+  dbInfo,
+}) {
+  const { name, protocol } = dbInfo;
+  const { hostname, port } = window.document.location;
+  const host = `${hostname}:${port || '443'}`;
+
+  return (
+    <>
```

Comment:
I don't see a reason to have this in a separate file. I was going to say something about `Manually.tsx` but i decided to let that one as is, but normally i avoid implementing to accommodate future addons like `Automatically`, b/c it may or may not happen.